app/components/works/embargo_component.html.erb

```diff
@@ -11,20 +11,7 @@
       "Deposit" at the bottom of this page. Or you may specify a specific date in the future.</p>
 
     <%= render Works::AvailableDateComponent.new(form: form) %>
-    <% if user_can_set_access? %>
```

Comment:
I rather prefer to keep these conditionals here because then the AccessComponent doesn't need to know anything about availablilty.

Reply:
I can see that. It felt like a lot of duplication, but happy to revert this change.